tests/unit-tests/test_nn_model.py

```diff
@@ -515,6 +516,268 @@ def test_rgcn_with_edge_features(input_dim, output_dim, dev):
     desired_2 = th.ones(dst_idx.shape[0], output_dim) * (1 + 2*(2**-0.5))
     assert_almost_equal(actual_2.numpy(), desired_2.numpy(), decimal=5)
 
+def init_hgtlayer(layer):
+    """ Initialize an HGT layer to make it having all 1s weights, and all 0s biases.
+    """
+    for name, para in layer.named_parameters():
+        if 'bias' in name:
+            th.nn.init.constant_(para, 0)
+        else:
+            th.nn.init.constant_(para, 1)
+
+@pytest.mark.parametrize("input_dim", [32])
+@pytest.mark.parametrize("output_dim", [32,64])
+@pytest.mark.parametrize("dev", ['cpu','cuda:0'])
+def test_hgt_with_edge_features(input_dim, output_dim, dev):
+    """ Test the RelGraphConvLayer that supports edge features.
```

Comment:
Can we create a test case where the `subg` only has two nodes and one edge or three nodes with two edges, so that we can compute the exact computation result of HGTLayerwithEdgeFeat.forward?

Reply:
Yes, add new test cases.